packages/plugins/plugin-braze-middleware/README.md

```diff
@@ -0,0 +1,67 @@
+# @segment/analytics-react-native-plugin-braze-middleware
+
+`BeforePlugin`  to debounce `identify` events for [Braze](https://www.braze.com).
```

Comment:
Let's add that this is for people who are using `Braze` through cloud mode, and also have a link for the Braze device mode plugin here. (and do the reverse in the Device Mode plugin)

Reply:
Also add that you cannot have both in the same instance.